src/test/java/TicTacToe/TicTacToeTest.java

```diff
@@ -0,0 +1,109 @@
+package TicTacToe;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+import static org.mockito.Mockito.*;
+
+import org.mockito.MockitoAnnotations;
+
+public class TicTacToeTest {
```

Comment:
Zamiast wielu testów, zrobiłbym jeden w rmaach, któego przetestowałbym:
- w tury i koniec gry
- Czy gracze naprzemiennie stawiają CIRCLE i CROSS
- Czy gra się kończy jak WinLogic powie, że ma się skończyc

Moim zdnaiem to by starczyło. Nie rozumiem poc ot e wszystkie testy.